fmn/consumer/backends/mail.py

```diff
@@ -84,8 +86,8 @@ def send_mail(self, session, recipient, subject, content,
         # Explicitly declare encoding, but remove the transfer encoding
         # https://github.com/fedora-infra/fmn/issues/94
         email_message.set_charset('utf-8')
-        if 'Content-Transfer-Encoding' in email_message:
-            del email_message['Content-Transfer-Encoding']
+        #if 'Content-Transfer-Encoding' in email_message:
+        #    del email_message['Content-Transfer-Encoding']
```

Comment:
It should just be sufficient to uncomment these lines as the Content-Transfer-Encoding was originally set to ``base64``.

I wasn't explicitly encoding things as base64 here, so I thought it was a mistake.  Clearly, that was wrong.